get_github_coderabbit_review/coderabbit_fetcher/cache/redis_cache.py

```
@@ -0,0 +1,386 @@
+            )
+
+            self._redis_client = redis.Redis(connection_pool=connection_pool)
+
+            # Test connection
+            self._redis_client.ping()
+            self._available = True
+
+            logger.info(f"Connected to Redis at {self.config.host}:{self.config.port}")
+
+        except ImportError:
+            logger.error("Redis library not installed. Install with: pip install redis")
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)